crates/oxide/src/parser.rs

```diff
@@ -240,7 +240,11 @@ impl<'a> Extractor<'a> {
     }
 
     #[inline(always)]
-    fn is_valid_candidate_string(candidate: &'a [u8]) -> ValidationResult {
+    fn is_valid_candidate_string(
+        candidate: &'a [u8],
+        input: &[u8],
+        start_idx: usize,
+    ) -> ValidationResult {
```

Comment:
Not too crazy about this having to take in the entire input _and_ the candidate slice. Not a big deal but maybe something to revisit eventually.

Reply:
Can probably even get rid of that code again, not sure. This essentially validates if it is preceded by `var(` to prevent false positives like `{ '--color-red-500': 'bar' }` where otherwise `--color-red-500` might be considered "a used variable".

Another situation is for custom-idents that aren't actual CSS variables… but maybe that's fine?